src/autoTune.ml

```diff
@@ -216,6 +216,7 @@ let focusOnSpecification () =
   | NoDataRace -> (*enable all thread analyses*)
     print_endline @@ "Specification: NoDataRace -> enabling thread analyses \"" ^ (String.concat ", " notNeccessaryThreadAnalyses) ^ "\"";
     enableAnalyses notNeccessaryThreadAnalyses;
+  | NoTermination -> ()
```

Comment:
Maybe enable your termination analysis and one of the relational analyses here?

Reply:
@Nimmseasy took care of this.